gulpfile.js

```diff
@@ -1328,7 +1328,7 @@ function compileJs(srcDir, srcFilename, destDir, options) {
             } else if (destFilename === 'amp-esm.js') {
               return enableLocalTesting(unminifiedRuntimeEsmTarget);
             } else if (destFilename === 'amp4ads-v0.js') {
-              return enableLocalTesting(unminifiedRuntimeEsmTarget);
+              return enableLocalTesting(unminifiedAdsTarget);
```

Comment:
Sheesh! This must've been me. Nice catch.

Reply:
And I was wrong yet again >_<

See https://github.com/ampproject/amphtml/commit/5fd7c8a3632391356ad1c1b2afbd2b652fc28d5c#r33407960